integration-tests/src/test/scala/tech/beshu/ror/integration/suites/RemoteReindexSuite.scala

```diff
@@ -74,36 +75,36 @@ trait RemoteReindexSuite
       "request specifies source and dest index that are allowed on both source and dest ES" in {
```

Comment:
we'd rather merge suites then split them. That's why it'd be nice to merge these two reindex suites. Moreover, according to the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs.html), I see that ES classifies reindex operation as one of Document API operations. So maybe we can extend `DocumentApiSuite`?

Reply:
ok, but maybe let's do it in one of next PRs? I touched Reindex suites because I wanted to change response code matching according to what I've done in `ResizeSuite`

Reply:
ok